lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 use crate::errors::invalid_data_error;
+use crate::util::to_unicode;
```

Comment:
I believe it's more idiomatic to `use crate::util` and then refer to it as `util::to_unicode`.

https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths